deploy/kubernetes/images/prow-gke-release-staging-rc-1-19/image.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: builtin
+kind: ImageTagTransformer
+metadata:
+  name: imagetag-csi-provisioner-prow-rc
+imageTag:
+  name: k8s.gcr.io/sig-storage/csi-provisioner
+  newTag: "v2.1.0"
+---
+
+apiVersion: builtin
+kind: ImageTagTransformer
+metadata:
+  name: imagetag-csi-resize-prow-rc
+imageTag:
+  name: k8s.gcr.io/sig-storage/csi-resizer
+  newTag: "v1.1.0"
+---
+
+apiVersion: builtin
+kind: ImageTagTransformer
+metadata:
+  name: imagetag-csi-snapshotter-prow-rc
+imageTag:
+  name: k8s.gcr.io/sig-storage/csi-snapshotter
+  newTag: "v3.0.3"
```

Comment:
The snapshotter is v4 in 1.19, do we not want to use that here?

Reply:
from the release page I see 4.0.0 has min 1.20 hence I didnot include in 1.19.
https://github.com/kubernetes-csi/external-snapshotter/releases/tag/v4.0.0